### PR TITLE
magnus works with gfortran compiler

### PR DESCRIPTION
**Pull request by gonzalodm**
I have changed TD.f by TD.f90 and I corrected alpha value (second argument) in cublas_zaxpy function.
@charlyqchm, I think you should work on this TD.f90

### CODE REVIEW

**Comment by fedepedron** [comment]
For the record. Tested with LIO/GROMACS with/without AINT and full cpu. I still need additional tests and review the files.

File: lioamber/TD.f90 [hunk 1]
```diff
@@ -787,12 +807,12 @@ SUBROUTINE TD()
              call g2g_timer_stop('complex_rho_on_to_ao')
 #endif
 !       rho1=REAL(rho1)
-c with matmulnanoc:
-c (NO LONGER AVAILABLE; USE BASECHANGE INSTEAD)
-c          call matmulnanoc(rho,xtrans,rho1,M)
-c          rho1=basechange(M,X,rho,Xtrans)
-c          rho1 = REAL(rho1)
-c The real part of the density matrix in the atomic orbital basis is copied in RMM(1,2,3,...,MM) to compute the corresponding fock matrix.
+! with matmulnanoc:
```
Comment (fedepedron) [inline lioamber/TD.f90]: Commented code should be deleted if it does not provide additional functionality.

**Comment by ramirezfranciscof** [comment]
Is this PR still relevant or does #122 now includes this?

**Comment by gonzalodm** [comment]
I think no, The PR #122 includes this.!!!!

2017-08-18 18:10 GMT-03:00 ramirezfranciscof <notifications@github.com>:

**Comment by ramirezfranciscof** [comment]
Ok, then I'll be closing this. Should anything happen, I think we can reopen it.

**Pull request closed without merging**